app/views/buys/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for :head do %>
+  <%= stylesheet_link_tag 'category_index' %>
+<% end %>
+
+<% content_for :page_name, "New Transaction" %>
+
+<%= render 'helper/navbar' %>
+
+<div class="container-fluid">
+  <%= form_with(model: [@author, @category, @buy], url: buys_path, method: :post) do |f| %>
+    <div class="mt-5 mb-3">
+      <%= f.text_field :name, class: "form-control", id: "name", placeholder: "Name" %>
+    </div>
+    <div class="mb-3">
+      <%= f.number_field :amount, class: "form-control", id: "amount", placeholder: "Amount" %>
+    </div>
+    <div class="mb-3">
+      <p>Categories (please select at least one): </p>
+      <div class="d-flex flex-wrap">
+        <% Category.all.each do |category| %>
+          
+            <div class="form-check mx-3">
+              <%= check_box_tag "buy[category_ids][]", category.id, @buy.category_ids.include?(category.id), class: "form-check-input" %>
+              <%= label_tag "buy_category_ids_#{category.id}", category.name, class: "form-check-label" %>
+            </div>
+          
+        <% end %>
+      </div>
+    </div>
+    <button type="submit" class="btn btn-success" style="width: 100%;">Save Transaction</button>
+    <div class="d-flex justify-content-center mt-4">
+      <%= link_to 'Back', category_path(id: @cat_id), class: 'btn btn-warning', style: "width: 100%;" %>
+    </div>
```

Comment:
- You have done well so far, but please make sure your back (<) button is designed and placed as it is placed in the design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please fix this 👍 👍 
- Also, please ensure that the `Save Transaction` button follows the color code i.e `#5fb523` and is aligned to the bottom of the page as I pointed out for the button in the `category#index` page. Please fix this 👍 👍 
Design template:
![bajlntshaet](https://github.com/AliRezaBmeDu/budget-app/assets/73607512/4b219610-0499-4776-b587-8561a9bfaef4)

Your project:
![wfqfqwf](https://github.com/AliRezaBmeDu/budget-app/assets/73607512/29f047cf-92e7-42cd-a26c-3c3c9d033c93)

Reply:
done